docs/VARIABLES.md

```diff
@@ -2380,6 +2380,86 @@ Default:  "{{mds_ssl_enabled}}"
 
 ***
 
+### sso_mode
+
+SSO mode for C3. Possible values: oidc. if enabling oidc you must set  sso_groups, sso_sub, sso_jwks_uri, sso_authorize_uri, sso_token_uri, sso_issuer_url, sso_client_id, sso_client_password in MDS
+
+Default:  none
+
+***
+
+### sso_groups
+
+Groups in JWT
+
+Default:  none
+
+***
+
+### sso_sub
```

Comment:
Similar to groups, I feel `claim` / `claim_name` should be added.

Reply:
do you mean sso_claim_sub?

Reply:
I would suggest `sso_sub_claim`.